products/stratos/modules/distribution/src/main/conf/drools/mincheck.drl

```diff
@@ -83,9 +81,8 @@ dialect "mvel"
                                  0, 0, 0, 0, 0, 0,0, 0, 0,
                                  clusterInstanceContext.getMinInstanceCount(), 0,
                                  additionalInstances, scalingReason);
-        } else {
-            log.warn("Scaling decision publisher is not enabled");
         }
+
```

Comment:
@swgkg Because if we are not enabling metering and monitoring feature, this logs will be getting accumulated for each thrift events related to metering and monitoring and it will be a unwanted overhead.